### DISCUSSION TITLE
WebRTC Additional Configuration Clarification

### DISCUSSION THREAD

**Discussion by crazytimmy** [mrlt8/docker-wyze-bridge#747] [Q&A] [answered]
Hi,
I'm referencing the WebRTC section in the readme.md file: https://github.com/mrlt8/docker-wyze-bridge#webrtc

"WebRTC should work automatically in Home Assistant mode, **however, some additional configuration is required to get WebRTC working in the standard docker mode.**"

Are the additional ports & WB_IP required to view the feed's on the web interface? I didn't configure these initially and it seems to have worked so I'm a bit confused as to why this section suggests additional configuration is required in standard docker mode.

Can anyone tell me what I'm missing?

**Comment by mrlt8** [comment, marked as answer]
Docker containers are isolated from the host so we can't see the host's IP or the port configuration. However, when run as a Home Assistant add-on, there's a [Supervisor API](https://developers.home-assistant.io/docs/api/supervisor/endpoints) that allows us to grab the IP from inside the container to do the necessary configuration for WebRTC.

**Reply by crazytimmy** [reply]
Thanks for the clarification I think I understand.

As I'm not using Home Assistant and running this through a Docker container to view through the Wyze-Bridge WebUI I should need to add in these environment variables for WebRTC to function.

This is what confused me when I initially ran the docker container and didn't add these additional environment variables the WebRTC streams still seemed to work on the main page. Adding the environment variables didn't seem to make any noticeable functional improvements on the main page.

Thanks for your time!

**Reply by mrlt8** [reply]
The WebUI will fallback to HLS if WebRTC is not available, so that may be what you are seeing.

You should also see "WebRTC" listed under the streams for each camera if it is enabled.
